### DISCUSSION TITLE
Move all discussions to iptv-org.github.io repository

### DISCUSSION THREAD

**Discussion by freearhey** [iptv-org/community#523] [Polls]
When the [iptv-org/iptv](https://github.com/iptv-org/iptv) repository was last locked, the [Discussions](https://github.com/iptv-org/iptv/discussions) were also inaccessible and we had to open separate [Discussions](https://github.com/orgs/iptv-org/discussions) in the [iptv-org/iptv-org.github.io](https://github.com/orgs/iptv-org/discussions) repository. 

Given that the lock could happen again at any time I would like to move all topics to the iptv-org.github.io repository so we don't lose them again.

**Comment by Emilselimli** [comment]
good idea
